packages/url_launcher/example/lib/main.dart

```diff
@@ -138,13 +151,19 @@ class _MyHomePageState extends State<MyHomePage> {
               }),
               child: const Text('Launch in app'),
             ),
-            const Padding(padding: EdgeInsets.all(16.0)),
             RaisedButton(
               onPressed: () => setState(() {
                 _launched = _launchInWebViewWithJavaScript(toLaunch);
               }),
               child: const Text('Launch in app(JavaScript ON)'),
             ),
+            RaisedButton(
```

Comment:
nit: This extra item is now causing the bottom to overflow in my testing. We should probably nest these all in a `ListView` now.

Reply:
Hi, I've done this, please take a look, thanks~